File: Assistant/lib/view/modding_worker/setting.dart
import '/common.dart';

// ----------------

class Setting {
  String       kernel;
  String       script;
  List<String> argument;
  Boolean      immediateLaunch;
  List<String> messageFont;
  Setting({
    required this.kernel,
    required this.script,
    required this.argument,
    required this.immediateLaunch,
    required this.messageFont,
  });
}

File: AssistantPlus/Application/View/ModdingWorker/Setting.cs
#pragma warning disable 0,
// ReSharper disable

using AssistantPlus;
using AssistantPlus.Utility;

namespace AssistantPlus.View.ModdingWorker {

	public record Setting {
		public String       Kernel          = default!;
		public String       Script          = default!;
		public List<String> Argument        = default!;
		public Boolean      AutomaticScroll = default!;
		public Boolean      ImmediateLaunch = default!;
		public String       MessageFont     = default!;
	}

}
